ScienceJournal/Sensors/MagnetometerSensor.swift

```diff
@@ -64,12 +65,44 @@ class MagnetometerSensor: MotionSensor {
     guard let magneticField =
         MagnetometerSensor.motionManager.deviceMotion?.magneticField.field else { return }
 
+    guard isValid(magneticFieldReading: magneticField) else {
```

Comment:
Any reason this guard couldn't be combined with the guard above?

Reply:
I didn't like how it looked, that is all. But I think you're right, it's better to combine.